plasticos_web_leads/models/web_lead.py

```
@@ -0,0 +1,427 @@
+                        "confidence": 0.7
+                    },
+                    "material": { ... },
+                    "observations": [ ... ]
+                }
+            }
+
+        Returns the created web.lead record.
+        """
+        lead_id = payload.get("lead_id")
+        if not lead_id:
+            raise UserError("Missing required field: lead_id")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)